kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +65,9 @@
   public static final String SSEA_CONFIG = "s3.ssea.name";
   public static final String SSEA_DEFAULT = "";
 
+  public static final String SSE_AWS_KMS_ID_CONFIG = "s3.sse.aws.kms.id";
+  public static final String SSE_AWS_KMS_ID_DEFAULT = "";
```

Comment:
The "aws" part of this configuration option seems redundant. "s3.sse.kms.id" (and renaming constants and methods here to match) would be more elegant. Not a blocker.

Reply:
Good point. Though you one can see 'aws' included in headers, it's better if we follow naming conventions in APIs such the one in `aws-cli`. 

Going with `s3.sse.kms.key.id`